peerconnection_go_test.go

```diff
@@ -43,6 +44,107 @@ func (api *API) newPair(cfg Configuration) (pcOffer *PeerConnection, pcAnswer *P
 	return pca, pcb, nil
 }
 
+const liteOffer = `v=0
+o=- 1613490992396387 1 IN IP4 123.45.67.89
+s=VideoRoom 5538594542389977
+t=0 0
+a=group:BUNDLE audio video
+a=msid-semantic: WMS janus
+a=ice-lite
+m=audio 9 UDP/TLS/RTP/SAVPF 111
+c=IN IP4 123.45.67.89
+a=sendonly
+a=mid:audio
+a=rtcp-mux
+a=ice-ufrag:tTFQ
+a=ice-pwd:FO9fjczpXrt4NbCcrlW3wK
+a=ice-options:trickle
+a=fingerprint:sha-256 4A:13:C9:00:E4:A9:40:0C:7B:09:A0:FB:3E:F7:C3:35:1A:C8:B9:26:3B:E4:EB:5B:2B:96:F6:34:4A:6E:24:3E
+a=setup:actpass
+a=rtpmap:111 opus/48000/2
+a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=rtcp-fb:111 transport-cc
+a=msid:janus janusa0
+a=ssrc:2666679401 cname:janus
+a=ssrc:2666679401 msid:janus janusa0
+a=ssrc:2666679401 mslabel:janus
+a=ssrc:2666679401 label:janusa0
+m=video 9 UDP/TLS/RTP/SAVPF 96 97
+c=IN IP4 123.45.67.89
+a=sendonly
+a=mid:video
+a=rtcp-mux
+a=ice-ufrag:tTFQ
+a=ice-pwd:FO9fjczpXrt4NbCcrlW3wK
+a=ice-options:trickle
+a=fingerprint:sha-256 4A:13:C9:00:E4:A9:40:0C:7B:09:A0:FB:3E:F7:C3:35:1A:C8:B9:26:3B:E4:EB:5B:2B:96:F6:34:4A:6E:24:3E
+a=setup:actpass
+a=rtpmap:96 VP8/90000
+a=rtcp-fb:96 ccm fir
+a=rtcp-fb:96 nack
+a=rtcp-fb:96 nack pli
+a=rtcp-fb:96 goog-remb
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
+a=extmap:12 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
+a=rtpmap:97 rtx/90000
+a=fmtp:97 apt=96
+a=ssrc-group:FID 542205910 3174992476
+a=msid:janus janusv0
+a=ssrc:542205910 cname:janus
+a=ssrc:542205910 msid:janus janusv0
+a=ssrc:542205910 mslabel:janus
+a=ssrc:542205910 label:janusv0
+a=ssrc:3174992476 cname:janus
+a=ssrc:3174992476 msid:janus janusv0
+a=ssrc:3174992476 mslabel:janus
+a=ssrc:3174992476 label:janusv0
+`
+
+// this test asserts that if an ice-lite offer is received, 
+// pion will take the ICE-CONTROLLING role
+func TestICELite(t *testing.T) {
+
+	SDP := SessionDescription{SDP:liteOffer,Type:SDPTypeOffer}
+
+
+	config := Configuration{
+		ICEServers: []ICEServer{
+			{
+				URLs: []string{"stun:stun.l.google.com:19302"},
+			},
+		},
+		SDPSemantics: SDPSemanticsUnifiedPlanWithFallback,
+	}
+
+	peerConnection, err := NewPeerConnection(config)
```

Comment:
I think we can just do `Configuration{}` this test isn't a PlanB/Unified-Plan issue.